packages/shared/src/styles/base.css

```diff
@@ -52,6 +52,7 @@ body {
   --theme-background-bun: theme('colors.bun.80');
   --theme-background-onion: theme('colors.onion.80');
   --theme-background-pepper: theme('colors.pepper.20');
+  --theme-background-cabbage-opacity-24: theme('colors.cabbage.50')24;
```

Comment:
Are you sure we need to introduce a new color, would think it should already exist.